server/MyMovieFlix/src/main/java/io/egen/movieflix/utils/ApplicationLogger.java

```diff
@@ -0,0 +1,22 @@
+package io.egen.movieflix.utils;
+
+import org.apache.log4j.Logger;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ApplicationLogger {
```

Comment:
`lombok` has a logger annotation that provides bridges for most of the logger implementations. Check that out.